### PR TITLE
Update lockingTextWriter docs

### PR DESCRIPTION
**Pull request by JackStouffer**
Add note about buffering.

### CODE REVIEW

File: std/stdio.d [hunk 1]
```diff
@@ -2981,8 +2981,16 @@ $(D Range) that locks the file and allows fast writing to it.
...
+     * out of scope.
+     *
      * Returns: An $(REF_ALTTEXT output range, isOutputRange, std, range, primitives)
-     * that locks the file and allows fast writing to it.
+     * which accepts string types, `ubyte[]`, individual character types, and
+     * individual `ubyte`s.
+     *
+     * Note: Writing either arrays of `char`s or `ubyte`s is much faster than
+     * writing each character individually a range. For large files,
+     * writing using an array acting as output buffer can result in a speed increase.
```
Comment (schveiguy) [inline std/stdio.d]: "individually a range" -> ??? Not sure what you meant here, but it needs rewording

"For large files, writing using an array acting as output buffer can result in a speed increase."

Isn't this redundant? You just said writing an array is "much faster". I don't get the point about "acting as an output buffer".

File: std/stdio.d [hunk 2]
```diff
@@ -2981,8 +2981,17 @@ $(D Range) that locks the file and allows fast writing to it.
...
+     * Output range which locks the file when created, and unlocks it when it goes
+     * out of scope.
+     *
      * Returns: An $(REF_ALTTEXT output range, isOutputRange, std, range, primitives)
-     * that locks the file and allows fast writing to it.
+     * which accepts string types, `ubyte[]`, individual character types, and
+     * individual `ubyte`s.
+     *
+     * Note: Writing either arrays of `char`s or `ubyte`s is faster than
+     * writing each character individually from a range. For large files,
```
Comment (jondegenhardt) [inline std/stdio.d]: Sentence talks about the speed increase writing to files resulting from writing in chunks. A minor clarification would be that this applies to writing to `stdout` as well as files.

Comment (schveiguy) [reply std/stdio.d]: How about just "large amounts of data"?

Comment (jondegenhardt) [reply std/stdio.d]: Sounds goods to me.

**Pull request merged**